flow-report/src/i18n/ui-strings.js

```diff
@@ -133,4 +133,8 @@ export const UIStrings = {
   }`,
   /** Label for a list of Lighthouse audits that are the most impactful. */
   highestImpact: 'Highest impact',
+  /** Label indicating that Lighthouse throttled the page to emulate a slow 4G network connection. */
+  runtimeSlow4g: 'Slow 4G',
```

Comment:
if you wanna put these in renderer.util that'd be great. i have some untranslated uses of those right now.

Reply:
i'm moving them.